.master-plan.yml

```diff
@@ -864,4 +864,12 @@ spec:
     labels:
       test/feature: "DIRECTOR PROFILE"
       test/tags: "DIRECTOR, PROFILE"
-      testplan: "https://e2e.mayadata.io/docs/director/authentication/TCID-GUI-PROFILE"
\ No newline at end of file
+      testplan: "https://e2e.mayadata.io/docs/director/authentication/TCID-GUI-PROFILE"
+
+  - tcid: TCID-DIR-GUI-OPENEBS-COMPONENTS-VERSION
+    name: "TCID-DIR-GUI-OPENEBS-COMPONENTS-VERSION"
+    description: "Test director OpenEBS components version"
+    labels:
+      test/feature: "DIRECTOR OPENEBS COMPONENTS VERSION"
+      test/tags: "DIRECTOR, OPENEBS, VERSION"
+      testplan: "https://e2e.mayadata.io/docs/director/openebs-provisioning/provisioning"
```

Comment:
I think this is not the right link

Reply:
i have put director openebs link because i was not able to find right link for that test case.